packages/astro/src/runtime/server/index.ts

```diff
@@ -546,10 +546,8 @@ export async function renderHead(result: SSRResult): Promise<string> {
 	const styles = Array.from(result.styles)
 		.filter(uniqueElements)
 		.map((style) => {
-			// TODO: clean up legacyBuild from metadata
-			const styleChildren = !result._metadata.legacyBuild ? '' : style.children;
 			return renderElement('style', {
```

Comment:
Is this hole .map() still needed?

Reply:
I was wondering that too. It looks like this converts each `style` (which is an `SSRElement` object) to an HTML element with `style.props` applied. I'd assume it's still necessary for that

Reply:
It's not needed

Reply:
Removed!